mixnode/src/commands/upgrade.rs

```diff
@@ -381,6 +381,32 @@ fn patch_0_10_2_upgrade(
     Ok(upgraded_config)
 }
 
+fn patch_0_10_3_upgrade(
```

Comment:
I think this should be put together to the `undetermined_version_upgrade` as technically we haven't made any release since 0.10.2

Reply:
Got it,  will put everything in that one until we make the next release.